examples/notebooks_new_field/introduction.ipynb

```diff
@@ -0,0 +1,1099 @@
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "class d0domain(SubDomain):\n",
+    "    name = 'd0'\n",
+    "    def define(self, dimensions):\n",
+    "        x, z = dimensions\n",
+    "        return {x: z, z: z}\n",
+    "d0_domain = d0domain()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "After defining the spatial parameters and *subdomains*, we generate the *spatial grid*:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "grid = Grid(origin=origin,extent=extent,shape=shape,subdomains=(d0_domain))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The velocity field is set bellow, representing four different layers:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "v0 = np.zeros((nptx,nptz))                     \n",
+    "p0 = 0    \n",
+    "p1 = int((1/4)*nptz)\n",
+    "p2 = int((1/2)*nptz)\n",
+    "p3 = int((2/3)*nptz)\n",
+    "p4 = nptz  \n",
+    "v0[0:nptx,p0:p1] = 1.5\n",
+    "v0[0:nptx,p1:p2] = 3.0\n",
+    "v0[0:nptx,p2:p3] = 5.0\n",
+    "v0[0:nptx,p3:p4] = 7.0"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To build the *temporal grid* we use the command *TimeAxis*. Before using  *TimeAxis*, we need to set the following parameters:\n",
+    "\n",
+    "- *t0* is the initial time simulation in milliseconds (ms);\n",
+    "- *tn* is the final time simulation in milliseconds (ms);\n",
+    "- *CFL* is the numerical value that imposes a restriction on the size of the time step, in order to obtain stability of the method;\n",
+    "- *vmax* denotes the maximum velocity propagation. This value is obtained by analyzing the maximum value of the *vel* array;\n",
+    "- *dtmax* is the upper limit for $\\Delta t$. This value is constructed from the *hx*, *hy* and *vmax* values.\n",
+    "- *ntmax* is the number of time steps required to reach time $t_ {F}$;\n",
+    "- dt0 is the value of$\\Delta t$ that can be used in the numerical method and that guarantees its stability;\n",
+    "\n",
+    "The parameters mentioned above are given by:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "t0 = 0.\n",
+    "tn = 20000.   \n",
+    "CFL = 0.4\n",
+    "vmax  = np.amax(v0) \n",
+    "dtmax = np.float64((min(hx,hz)*CFL)/(vmax))\n",
+    "ntmax = int((tn-t0)/dtmax)+1\n",
+    "dt0   = np.float64((tn-t0)/ntmax)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To generate the *temporal grid* we use the variables given above and allocate them to some local variables of the *TimeAxis* function:\n",
+    "\n",
+    "- The start time *t0* is assigned to the local variable *start*;\n",
```

Comment:
same

Reply:
The name of this variable is correct if we consider the context of numerical problem.